### ISSUE TITLE
Test name in Fast Pass has insufficient contrast against its background on mouse hover

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#870]
The following AI-windows mode(s) have this issue:
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![17 1](https://user-images.githubusercontent.com/22068434/88356412-92915d80-cd1c-11ea-8613-955c9d9b96ac.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in the release at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1382.01. Please regress/close at your convenience.

**Comment by iamrafan** [comment]
Validated the fixes in Canary Release: v1.1.1383.01, all 6 modes are compliant
